src/session/input.rs

```diff
@@ -77,7 +77,9 @@ impl<'v, const N: usize> From<[SessionInputValue<'v>; N]> for SessionInputs<'_,
 	}
 }
 
-/// Construct the inputs to a session from an array or named map of values.
+/// Construct the inputs ([`SessionInputs`]) to a session from either an array or a named map of values
+/// ([`SessionInputValue`]s). The crate provides conversion utilities for [`std::collections::HashMap`], [`Vec`],
+/// arrays and slices.
```

Comment:
Minor nit:
```diff
-Construct the inputs ([`SessionInputs`]) to a session
+Construct the inputs to a session ([`SessionInputs`])
```

"The crate provides conversion utilities..." should elaborate more. I assume this is talking about `SessionInputs`, in which case this detail should be documented (and perhaps demonstrated) in `SessionInputs` rather than the unrelated `inputs!` macro.

Reply:
My line of thinking was that many examples do `session.run(inputs![… … …])?;` and the `run` method takes an `impl Into<SessionInputs<'i, 'v, N>>` so it's not super obvious what to pass there. OTOH the  docs for `run` contains essentially the same info that I added in this PR to the `inputs!`-docs, so perhaps that's good enough? I removed it.